acquia_lift.links.action.yml

```diff
@@ -0,0 +1,5 @@
+entity.acquia_lift_slot.add_form:
+  route_name: entity.acquia_lift_slot.add_form
```

Comment:
Moving the this discussion to another thread which was focusing on "acquia_lift_slot" id.